src/components/input/multimodal/uploader/uploader.tsx

```diff
@@ -188,31 +207,45 @@ function Uploader(props: UploaderProps) {
         handleSuccessfulUpload(response.data, temporaryFileId)
```

Comment:
```suggestion
        handleSuccessfulUpload(response.data, newAddedFile.id)
```

Reply:
I think references to temporaryFileId in callbacks can be replaced with `newAddedFile.id`